ios/Exponent/Supporting/Info.plist

```diff
@@ -82,7 +85,18 @@
 	<array>
 		<string>waze</string>
 		<string>fbapi</string>
-		<string>fb-messenger-api</string>
+		<string>fbapi20130214</string>
+		<string>fbapi20130410</string>
+		<string>fbapi20130702</string>
+		<string>fbapi20131010</string>
+		<string>fbapi20131219</string>
+		<string>fbapi20140410</string>
+		<string>fbapi20140116</string>
+		<string>fbapi20150313</string>
+		<string>fbapi20150629</string>
+		<string>fbapi20160328</string>
+		<string>fbauth</string>
+		<string>fb-messenger-share-api</string>
```

Comment:
Do we still need https://github.com/expo/expo-cli/blob/6bacea668e954c24eb8d9ed6d46a5c46bc3e4a2f/packages/xdl/src/detach/IosNSBundle.ts#L317-L322? Should we `.uniq()` the array before building the app, add/remove some entries?

Reply:
hm so it seems the facebook docs are conflicting a little bit:
- when setting up the app it at the facebook developer console it says to include all of these (it know's we use FB SDK v5)
- but [here](https://developers.facebook.com/docs/ios/ios9/) it says that the `fbapi.*` schemes are only if you are on FB SDK v4.5 or older

I haven't noticed any issues when excluding the schemes, so probably safe to remove them